### PR TITLE
add pickems

### PR DESCRIPTION
**Pull request by stelar7**

### CODE REVIEW

File: src/ESports.ts [hunk 1]
```diff
@@ -18,22 +18,172 @@ interface ESportsLeagueSchedule {
...
+        for (const entry of leaderboard) {
+            const pickem = await this.getPickem(entry.vsUserId);
+            returnList.push(pickem);
+        }
+
+        return returnList;
+    }
+
+    public async getLeaderboard(): Promise<PickemLeaderboard> {
+        const data = await fetch("https://pickem.euw.lolesports.com/en-GB/api/get_vs_lists/series/5/user/4643536");
```
Comment (Querijn) [inline src/ESports.ts]: hardcoded values, can you make this configurable

File: src/ESports.ts [hunk 2]
```diff
@@ -18,22 +18,172 @@ interface ESportsLeagueSchedule {
...
+        return (await data.json())[0];
+    }
+
+    public async getPickem(id: number): Promise<PickemGroupPick> {
+        const url = "https://pickem.euw.lolesports.com/en-GB/api/get_group_picks/series/5/user/";
```
Comment (Querijn) [inline src/ESports.ts]: hardcoded values, can you make this configurable

File: src/ESports.ts [hunk 3]
```diff
@@ -18,22 +18,172 @@ interface ESportsLeagueSchedule {
...
     }
 
+    public async updateMemberList() {
+        this.currentList = await this.getMembersWithName();
+
+        setTimeout(this.updateMemberList.bind(this), 1000 * 60 * 60);
+    }
+
+    public async getCorrectPickem(): Promise<PickemGroupPick> {
+        const anyPick = await this.getPickem(4643536);
```
Comment (Querijn) [inline src/ESports.ts]: hardcoded values, can you make this configurable

File: src/ESports.ts [hunk 4]
```diff
@@ -18,22 +18,172 @@ interface ESportsLeagueSchedule {
...
             await this.loadData();
             this.postInfo();
+            this.updateMemberList();
         });
     }
 
+    public async updateMemberList() {
+        this.currentList = await this.getMembersWithName();
+
+        setTimeout(this.updateMemberList.bind(this), 1000 * 60 * 60);
```
Comment (Querijn) [inline src/ESports.ts]: hardcoded values, can you make this configurable

File: src/ESports.ts (hunk 2 above)
Comment (stelar7) [reply src/ESports.ts]: You want the entire url to be configurable?

**Comment by stelar7** [comment]
tested and working
https://i.imgur.com/IT4OZHj.png
https://i.imgur.com/2AfvjNY.png
https://i.imgur.com/pYrP9bY.png

**Comment by stelar7** [comment]
new leaderboard respecting equality
https://i.imgur.com/qdd4pzv.png

**Comment by stelar7** [comment]
Not an issue, i didnt know what the max points for the group stage is, so the correct choice gets infinite points.

To check an user use !pickem {your summoner name} (spaces allowed)

**Pull request merged**